scripts/dev.sh

```diff
@@ -52,6 +52,7 @@ openssl dhparam -out workspace/dhparam.pem 1024
 docker cp workspace/certificate.pem reverse-proxy:/etc/montagu/proxy/
 docker cp workspace/ssl_key.pem reverse-proxy:/etc/montagu/proxy/
 docker cp workspace/dhparam.pem reverse-proxy:/etc/montagu/proxy/
+docker cp $here/../2020 reverse-proxy:/usr/share/nginx/html
```

Comment:
Added this in to test serving the html page locally. What was your idea for adding in the real public tool data once it's available? Add it to the container manually or add it to the proxy repo to be deployed?

Reply:
ooh, adding it to the proxy is an interesting idea

Reply:
currently there's a script that adds it after deploy -  see last line here: https://github.com/vimc/montagu/blob/master/ReleaseProcess.md#deploying-to-uat--science